NEWS.md

```diff
@@ -0,0 +1,9 @@
+# NEWS
```

Comment:
Ah, I see you added NEWS.md here, so disregard my comment on PR 46!

Reply:
Yes, I started from here, and I will try to maintain it.